app/views/shared/_evidence_list.html.haml

```diff
@@ -36,6 +36,13 @@
               = document.created_at
             %td
               - if document.converted_preview_document.present?
-                = link_to t('common.view'), document_path(document), target: '_blank', class: 'view'
+                = link_to t('common.view'),
+                          document_path(document),
+                          target: '_blank',
+                          class: 'view',
+                          'aria-label': "Evidance: View document #{document.document_file_name}, opens in new window"
```

Comment:
@cleargif little typo 'Evidance' should read 'Evidence' (there are another occurrence of this few lines down).